src/shogun2-core/src/main/java/de/terrestris/shogun2/model/Plugin.java

```diff
@@ -52,6 +52,12 @@
 	@Type(type="text")
 	private String styleSheet;
 
+	/**
+	 * Whether the plugin is a system plugin or not. The properties and
+	 * source code of system plugins cannot be edited.
+	 */
```

Comment:
What about the following API  docs:

This can e.g. be used to disallow certain actions like editing of the source code. No such limitation will be enforced from SHOGun itself.